test/e2e/storage/drivers/csi.go

```diff
@@ -80,6 +81,9 @@ func initHostPathCSIDriver(name string, capabilities map[testsuites.Capability]b
 			SupportedFsType: sets.NewString(
 				"", // Default fsType
 			),
+			SupportedSizeRange: volume.SizeRange{
+				Min: "5Gi",
+			},
```

Comment:
> This can be removed. The hostpath driver is one of those drivers where we can be sure that it has no minimum size.

I am afraid the hostpath driver should have a minimum size 1Mb?
From source code here https://github.com/kubernetes-csi/csi-driver-host-path/blob/3488dc7f994e33485629b86b69a6f34ebb7ef2d9/pkg/hostpath/hostpath.go#L181 If the volume access type is 'block', then the driver will call `fallocate` with size unit M .
Too small size will be invalid.

Reply:
Right, I was thinking of filesystem mode. For block mode I am not sure whether the driver should round up internally - this sounds like a bug in the driver.

Anyway, let's then keep the minimum size.